DentOS_Framework/DentOsTestbed/src/dent_os_testbed/utils/test_utils/tb_utils.py

```diff
@@ -5,7 +5,9 @@
 import re
 import pytest
```

Comment:
Are you going to add a conftest.py with the new fixture in the scope of this PR?